### ISSUE TITLE
Create the schema for the Sanitization Quiz

### ISSUE THREAD

**Issue by xHergz** [xHergz/TheMakeupApp#16]
### Description:
Create the schema to store the sanitization quiz questions, answers, and user submitted responses.
### Tasks:
- [x] Create the Sanitization_Quiz_Question table
- [x] Add some initial sanitization questions
- [x] Create the Sanitization_Quiz_Answer table
- [x] Add some initial answers for the questions
- [x] Create the Sanitization_Quiz_Submission table
### Tests:
N/A

**Comment by xHergz** [comment]
Blocked by #15, needs the Artist_Application table for FK Reference

**Comment by xHergz** [comment]
No longer blocked by #15, doing dev in same branch.

**Comment by xHergz** [comment]
Diverting stored proc creating to another issue for when the design of the backend/page is figured out.
